### PR TITLE
variant on wires for inside a frame

### PR DESCRIPTION
**Pull request by chrisforbes**
Gets rid of z-fighting issues by having wires inside frames offset a bit.

en-wiring-interior-variant

### CODE REVIEW

**Comment by chrisforbes** [comment]
We should probably do something similar for entities placed inside frames. Likely not dedicated mesh pieces, but an offset along the normal.

File: assets/default.manifest [hunk 1]
```diff
@@ -133,6 +133,41 @@ assets = (
...
+    name = "wire_4i"
+    type = "mesh"
+    file = "mesh/wire_4i.obj"
+    scale = 0.03125
+    offset = [0.0,0.0,0.0]
+},
+{
+    name = "wire_8i"
+    type = "mesh"
+    file = "mesh/wire_8i.obj"
```
Comment (RobotCaleb) [inline assets/default.manifest]: Why are these new meshes? Will offset not work?

Comment (chrisforbes) [reply assets/default.manifest]: I don't want to use offset for this. It's for coping with needing different origin conventions, not for nudging things around by 1-2 voxels. These meshes may well also diverge soon -- I haven't done the 'L' filler pieces etc yet.

Comment (chrisforbes) [reply assets/default.manifest]: Potential confusion with what I'd said in the toplevel comment: for entities i want the /entity/ to be placed differently -- not an asset manager level variant.

**Pull request merged**